manifests/minio/minio.yaml

```diff
@@ -0,0 +1,135 @@
+# NOTE:
+# It is necessary to first deploy a secret that minio will use
+# for retrieving the username and password. To generate it
+# execute:
+# $ kubectl create secret generic minio-key --namespace kubeless --from-literal=accesskey=<username> --from-literal=secretkey=<password>
+---
+kind: PersistentVolumeClaim
+apiVersion: v1
+metadata:
+  name: minio
+  namespace: kubeless
+  annotations:
+    volume.alpha.kubernetes.io/storage-class: standard
```

Comment:
I think alpha storageclass is older than the k8s versions we claim to support with kubeless - and this should be removed (alpha behaved differently to beta-and-later wrt defaults).